### ISSUE TITLE
CrossDocked2020 dataset questions

### ISSUE THREAD

**Issue by juliabuhmann** [gnina/models#8]
Thanks a lot for making those datasets available! Very much appreciated.

1. There is no equation given in the paper on how you got from the Kd(?)/Ki(?) values provided by the PDBBind webpage to the pK values used in the paper and provided in the types-files.
2. Are both Kd and Ki values mapped to the same pK?
3. To understand the dataset better, I checked the pk-Values of this types-file: it0_tt_0_train0.types (from this directory: http://bits.csb.pitt.edu/files/crossdock2020/CrossDocked2020_types.tar.gz) 
  - I find that ~50% of the lines/samples have a pK value of 0. Is this meaningful?
  - I find (while FigureS12 is showing experimental pK values in the range of 2 -12), that ~30% of the pK values in the above types-files are negative

Thanks for clarification!

**Comment by dkoes** [comment]
pK is the negative log (base 10).  So nanomolar (10^-9) is 9.

We treat Kd/Ki/IC50 the same in mapping to pK. 

A 0 pK means there is no affinity data for that protein-ligand pair.  A negative pK means there is affinity data, but the pose is bad (>2 RMSD).  We apply a hinge loss in this case so that overpredicting the affinity of a bad pose is penalized but underpredicting is not.

**Comment by Kerro-junior** [comment]
I have a question, how does the value's sequence of types file be set? I guess it differs in every model, right?
where can I find this sequence setting for a specific model such as default2018?

**Comment by dkoes** [comment]
Every atom has an inherent gnina type, and these get aggregated into combined types (one on each line) in files like this: https://github.com/gnina/models/blob/master/acs2018/completelig which is included from the moldatalayer.

**Comment by Kerro-junior** [comment]
I have a question, what is the `stratify_receptor`? I couldn't find it in any of your papers. Is it sothing to do with binding affinity?

**Comment by dkoes** [comment]
Sample uniformly across receptors.  So if receptor A has 1000 poses and receptor B has 10 poses, if you have a batch of size 10 you would have 5 examples from A and 5 from B.  This avoids learning biases related to imbalances in the training data.
